packages/comments/src/lib/EditView.tsx

```diff
@@ -60,9 +67,41 @@ export function EditView({
     setInputText(retreived || '');
   }, [file, parentId, existingComment]);
 
-  function getNewComment(): Comment {
+  async function getCurrentSlideUri(courseId: string, sectionId: string) {
+    const response = await axios.get('/api/get-slides', {
```

Comment:
there is not need of any calls. you should get the uri in `course-view/[courseId].tsx`